### PR TITLE
Add API Metrics Reporting + Widget to VTv3

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
We added the API Metrics Reporting feature to all enrichment commands to allow customers to have greater insight into their API quota usage.

## Screenshots
Screen Shot 2022-06-06 at 18 15 14
Screen Shot 2022-06-07 at 11 33 03

## Minimum version of Cortex XSOAR
- Officially supported from XSOAR version 6.8.0 +

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/VirusTotal/ReleaseNotes/2_4_3.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+##### VirusTotal (API v3)
+- Added support for API Metrics Reporting to the  ***url***, ***domain***, ***file***, and ***ip*** commands.
```
Comment (bakatzir) [inline Packs/VirusTotal/ReleaseNotes/2_4_3.md:4]: mention 6.8.0?
```suggestion
- Added support for API Metrics Reporting to the  ***url***, ***domain***, ***file***, and ***ip*** commands which is usable from version 6.8.0.
```

**Review by bakatzir — COMMENTED** [review]
I also suggest reference in the readme of the integration that it supports API rate limiting fromversion 6.8.0.

**Review by danipv — APPROVED** [review]
Awesome feature. LGTM.

File: Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py [hunk 2]
```diff
@@ -1553,17 +1554,32 @@ def ip_command(client: Client, score_calculator: ScoreCalculator, args: dict, re
...
+                continue
         except Exception as exception:
             # If anything happens, just keep going
             demisto.debug(f'Could not process IP: "{ip}"\n {str(exception)}')
+            execution_metrics.general_error += 1
             continue
+        execution_metrics.success += 1
         results.append(
-            build_ip_output(client, score_calculator, ip, raw_response, argToBoolean(args.get('extended_data')))
+            build_ip_output(client, score_calculator, ip, raw_response, argToBoolean(args.get('extended_data'))).to_context()
```
Comment (yuvalbenshalom) [inline Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py:1574]: why did you add call to_context() here?

**Review by yuvalbenshalom — COMMENTED** [review]
see my comments.

File: Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py (hunk 2 above)
Comment (amshamah419) [reply Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py:1574]: There's a design issue with `return_results` where if you return an array of items (`CommandResults` or any of the new formats) It will return the object instead of the formatted version for the server.

Comment (amshamah419) [reply Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py:1574]: I can add a fix for it in this PR or open another one specifically for the CSP issue.

**Review by yuvalbenshalom — APPROVED** [review]
